scripts/release_scripts/get_repo_specific_changes.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that provides changes specific to oppia repo to be written
+to release summary file.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+from scripts import common
+
+GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING = 'git diff --name-only %s %s'
+GIT_CMD_SHOW_FORMAT_STRING = 'git show %s:feconf.py'
+VERSION_RE_FORMAT_STRING = r'%s\s*=\s*(\d+|\.)+'
+FECONF_VAR_NAMES = ['CURRENT_STATE_SCHEMA_VERSION',
+                    'CURRENT_COLLECTION_SCHEMA_VERSION']
+FECONF_FILEPATH = os.path.join('', 'feconf.py')
+
+
+def get_changes_in_versions(current_release):
+    """Checks if the versions for the exploration or collection schemas have
+    changed.
+
+    Args:
+        current_release: str. The current release tag to diff against.
```

Comment:
current_release_tag (the name should describe the thing it's naming, which in this case is a "tag"?)

Also what does "current" mean in this context? It seems ambiguous -- it could refer to the release the script is currently being run in the context of, right? Does this function need to know it's current -- should we use something like release_tag_to_diff_against instead?

Reply:
Yes, updated.